src/components/ebay-bar-chart/component.js

```diff
@@ -0,0 +1,292 @@
+                        ) {
+                            x =
+                                chartPosition.left +
+                                this.chart.chartWidth -
+                                hoverPoint.dlBox.width -
+                                6;
+                        }
+                        return { x, y };
+                    },
+            },
+            plotOptions: {
+                series: {
+                    description: this.input.description,
+                },
+                column: {
+                    events: {
+                        legendItemClick: function () {
+                            if (_this.input.stacked) {
+                                setTimeout(() => {
+                                    let topFound = false;
+                                    let bottomFound = false;
+
+                                    for (let i = 0; i < this.chart.series.length; i++) {
+                                        if (!bottomFound && this.chart.series[i].visible) {
+                                            this.chart.series[i].options.bottom = true;
+                                            bottomFound = true;
+                                        } else {
+                                            this.chart.series[i].options.bottom = false;
+                                        }
+                                    }
+
+                                    for (let i = this.chart.series.length - 1; i >= 0; i--) {
+                                        if (!topFound && this.chart.series[i].visible) {
+                                            this.chart.series[i].options.top = true;
+                                            topFound = true;
+                                        } else {
+                                            this.chart.series[i].options.top = false;
+                                        }
+                                    }
+                                    this.chart.redraw();
+                                }, 0);
+                            }
+                        },
+                    },
+                    className: this.input.stacked
+                        ? 'ebay-bar-chart-column-stacked'
+                        : 'ebay-bar-chart-column',
+                    stacking: this.input.stacked ? 'normal' : null,
+                    borderWidth: 1,
+                    borderColor: '#FFFFFF',
+                    borderRadius: !this.input.stacked ? 3 : null,
+                    groupPadding: 0.1,
+                    pointPadding: 0.15,
+                    states: {
+                        inactive: {
+                            opacity: 1,
+                        },
+                    },
+                    point: {
+                        events: {
+                            mouseOver: function () {
+                                const refPoint = this;
```

Comment:
This is a great example of what I was referring to in my previous comment about caching `this` to a meaningful const name. I would go bit further to explain what `refPoint` is, either make the const even more meaningful or add a comment.